examples/generated-complete/counter/impl.go

```
@@ -27,21 +27,36 @@ func (a *Counter) Type() string {
 // getCurrentValue retrieves the current counter value from actor state
 func (a *Counter) getCurrentValue(ctx context.Context) (int32, error) {
 	var value int32
-	err := a.GetStateManager().Get(ctx, stateKeyValue, &value)
+	
+	// Check if state manager is available
+	stateManager := a.GetStateManager()
+	if stateManager == nil {
+		// If state manager is not available, return default value
+		return 0, nil
```

Please return an error when the state manager is not available, and handle the error on the caller's side.
We'll add an error response type to the 200 response later, so for now, a workaround can be acceptable to some degree.